ephios/plugins/questionnaires/templates/questionnaires/savedanswer_form.html

```diff
@@ -0,0 +1,32 @@
+{% extends "core/settings/settings_base.html" %}
+{% load crispy_forms_filters %}
+{% load i18n %}
+
+{% block title %}
+    {% translate "Edit saved questionnaire answer" %}
+{% endblock %}
+
+{% block settings_content %}
+    <h3>
+        {% translate "Edit saved questionnaire answer" %}
+    </h3>
+
+    <h4>
+        {{ savedanswer.question.question_text }}
+    </h4>
+
+    {% if savedanswer.question.description %}
+        <p>
+            {{ savedanswer.question.description }}
+        </p>
+    {% endif %}
+
+    <form method="post">
```

Comment:
maybe we should display a warning that this does not change the answer for already existing participations?

Reply:
Added in dced00e1ba1b988f93ff581c9fedcb6b6f006c34